### ISSUE TITLE
Events / Pride Month

### ISSUE THREAD

**Issue by Simplyjoshie** [pokeclicker/pokeclicker#2444]
I believe the game needs more events especially for times like pride moths, black history month and etc

**Current Behavior**
The game is working fine in relation to this suggestion

**Improved Behavior**
I think more events especially for those real world events and times that matter a lot to people around the world. Id even maybe make it easier for people to put ideas forward for events if needing help with what pokemon to add. I personally would love to see more or all pride flag pokemon or support later but to start progress pride flag is the way to go

**Reason for change**
Will give people who are partially inactive a reason to make sure they are keeping in touch with the game more

**Detailed Description**
I am OBSESSED with playing pokeclicker lately and I've done most of the story I can so I'm trying to finish everything else that is available (no complaints i think you guys are great). So when looking over all else i can do in my main game I looked into events and realised that there are only a small few over the whole year. Although I can't say i know how hard it is to program an event I really think the game would greatly benefit with a few more AND although we are half way through I really think there should be a pride event. Now I do believe things like this are extremely important events and could have a lot in them i think we should have a Progress Pride Flag Vivillion for even a week at the end of them month.
I also think it would be cool to also support things like black history month with pokemon dressed as black historical figures or paying homage to.

**Comment by RedSparr0w** [comment]
Closing as per comment here:
https://github.com/pokeclicker/pokeclicker/issues/1661#issuecomment-982377178

**TLDR:**
- Don't want the game flooded with event Pokémon.
- There's a lot of days/months for a lot of things, don't want to have to gate keep what gets in and what doesn't.

**Comment by Simplyjoshie** [comment]
> 

All fair to not want to flood a game with event Pokemon. But when it comes to not wanting to say you're gate keeping I think that would be fair if you were picking holidays that are celebrated by less than half the world like Halloween but when it comes to major events, times and holidays that try to support equality and even freedom for many I think the only people who would complain that Pride or Black History/Reconciliation/etc that recognises racist history and promotes equality and learning of erased pasts would be people who think these events are nothing or useless (eg. homophobes, transphobes, racists and etc). So i dont think that second point of gate keeping is at all relevant when it comes to important events like these
Not trying to sound rude just needed to make the point clear
